pkg/storage/storage.go

```diff
@@ -0,0 +1,73 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type Profile struct {
+	Name string `json:"name"`
+	Key  string `json:"key"`
+}
+
+type Storage struct {
+	Default     string     `json:"default"`
+	Profiles    []*Profile `json:"profiles"`
+	StoragePath string     `json:"-"`
+}
+
+func (s *Storage) checkPath() error {
+	if _, err := os.Stat(s.StoragePath); os.IsNotExist(err) {
+		return err
+	}
+	return nil
+}
+
+func (s *Storage) Load() error {
+	if err := s.checkPath(); err != nil {
+		return err
+	}
+
+	content, err := os.ReadFile(s.StoragePath)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(content, s)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Using decoder interface is preferable since it's at least twice as lower on memory usage and don't actually know the file size